debian/usr.bin.snap-confine

```diff
@@ -112,6 +114,15 @@
     mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
     mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
     mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    mount options=(rw bind) /snap/ubuntu-core/*/etc/alternatives/ -> /etc/alternatives/,
+
+    # This is a somewhat of a blank check for mount profiles.  Mount profiles
+    # are written by the trusted snapd to a root-writable location and
+    # processed by snap-confine. Regardless of configuration options and
+    # runtime factors they are processed after pivot_root (on classic systems)
+    # so snap.rootfs is not a factor.
+    mount options=(rw bind),
+    mount options=(ro bind),
```

Comment:
We control mount points via interfaces but we allow any values to be used AFAIR

I'd like to make --enable-rootfs-is-core-snap used everywhere with this upload. The other option should be entirely removed as it just adds code that isn't used or tested.

As for mounting things on /var, isn't that something that will only be visible to the process? The outside world won't see the /var bind mount, will it?

Perhaps we should write a few spread tests, if you give me a few things that you'd like to ensure are working correctly I can make that happen.